2020-03-18 13:29:05.619  INFO 3312 --- [restartedMain] com.ykcoding.BlogApplication             : Starting BlogApplication on LAPTOP-3VL4VRID with PID 3312 (E:\coding\JavaFile\blog\target\classes started by Lenovo in E:\coding\JavaFile\blog)
2020-03-18 13:29:05.732 DEBUG 3312 --- [restartedMain] com.ykcoding.BlogApplication             : Running with Spring Boot v2.2.5.RELEASE, Spring v5.2.4.RELEASE
2020-03-18 13:29:05.732  INFO 3312 --- [restartedMain] com.ykcoding.BlogApplication             : The following profiles are active: dev
2020-03-18 13:29:05.936  INFO 3312 --- [restartedMain] o.s.b.devtools.restart.ChangeableUrls    : The Class-Path manifest attribute in C:\Users\Lenovo\.m2\repository\org\glassfish\jaxb\jaxb-runtime\2.3.2\jaxb-runtime-2.3.2.jar referenced one or more files that do not exist: file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.xml.bind-api-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/txw2-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/istack-commons-runtime-3.0.8.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/stax-ex-1.8.1.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/FastInfoset-1.2.16.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.activation-api-1.2.1.jar
2020-03-18 13:29:05.939  INFO 3312 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2020-03-18 13:29:05.941  INFO 3312 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2020-03-18 13:29:10.332  INFO 3312 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-03-18 13:29:10.629  INFO 3312 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 274ms. Found 4 JPA repository interfaces.
2020-03-18 13:29:20.236  INFO 3312 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2020-03-18 13:29:20.260  INFO 3312 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-03-18 13:29:20.261  INFO 3312 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.31]
2020-03-18 13:29:20.896  INFO 3312 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-03-18 13:29:20.897  INFO 3312 --- [restartedMain] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 14955 ms
2020-03-18 13:29:21.655  INFO 3312 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-03-18 13:29:21.865  INFO 3312 --- [restartedMain] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.12.Final
2020-03-18 13:29:22.285  INFO 3312 --- [restartedMain] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-03-18 13:29:22.833  INFO 3312 --- [restartedMain] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2020-03-18 13:29:23.091 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:23.126 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:23.126  INFO 3312 --- [Druid-ConnectionPool-Create-36258358] c.a.druid.pool.DruidAbstractDataSource   : {dataSource-1} failContinuous is true
2020-03-18 13:29:23.640 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:24.149 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:24.661 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:25.169 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:25.676 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:26.180 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:26.683 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:27.185 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:27.691 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:28.193 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:28.694 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:29.198 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:29.701 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:30.205 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:30.708 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:31.210 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:31.713 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:32.217 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:32.722 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:33.228 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:33.732 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:34.240 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:34.743 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:35.245 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:35.748 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:36.250 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:36.754 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:37.258 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:37.761 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:38.262 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:38.766 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:39.270 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:39.773 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:40.275 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:40.778 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:41.281 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:41.782 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:42.287 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:42.790 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:43.294 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:43.796 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:44.298 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:44.801 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:45.303 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:45.808 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:46.312 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:46.815 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:47.319 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:47.821 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:48.324 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:48.828 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:49.335 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:49.843 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:50.348 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:50.856 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:51.364 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:51.867 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:52.371 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:52.875 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:53.486 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:53.990 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:54.492 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:54.998 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:55.504 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:56.041 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:56.562 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:57.068 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:57.574 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:58.086 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:58.591 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:59.095 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:29:59.598 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:00.102 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:00.605 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:01.114 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:01.617 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:02.120 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:02.624 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:03.126 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:03.628 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:04.132 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:04.636 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:05.151 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:05.653 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:06.155 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:06.656 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:07.159 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:07.662 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:08.181 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:08.684 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:09.188 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:09.693 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:10.198 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:10.712 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:11.223 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:11.728 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:12.235 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:12.742 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:13.245 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:13.748 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:14.255 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:14.762 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:15.267 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:15.772 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:16.274 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:16.777 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:17.280 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:17.782 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:18.687 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:19.188 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:19.710 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:20.248 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:20.758 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:21.265 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:21.768 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:22.270 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:22.778 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:23.282 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:23.785 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:24.293 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:24.796 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:25.303 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:25.814 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:26.321 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:26.824 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:27.337 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:27.851 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:28.355 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:28.858 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:29.360 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:29.863 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:30:30.365 ERROR 3312 --- [Druid-ConnectionPool-Create-36258358] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:41.033  INFO 13848 --- [restartedMain] com.ykcoding.BlogApplication             : Starting BlogApplication on LAPTOP-3VL4VRID with PID 13848 (E:\coding\JavaFile\blog\target\classes started by Lenovo in E:\coding\JavaFile\blog)
2020-03-18 13:31:41.098 DEBUG 13848 --- [restartedMain] com.ykcoding.BlogApplication             : Running with Spring Boot v2.2.5.RELEASE, Spring v5.2.4.RELEASE
2020-03-18 13:31:41.099  INFO 13848 --- [restartedMain] com.ykcoding.BlogApplication             : The following profiles are active: dev
2020-03-18 13:31:41.199  INFO 13848 --- [restartedMain] o.s.b.devtools.restart.ChangeableUrls    : The Class-Path manifest attribute in C:\Users\Lenovo\.m2\repository\org\glassfish\jaxb\jaxb-runtime\2.3.2\jaxb-runtime-2.3.2.jar referenced one or more files that do not exist: file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.xml.bind-api-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/txw2-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/istack-commons-runtime-3.0.8.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/stax-ex-1.8.1.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/FastInfoset-1.2.16.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.activation-api-1.2.1.jar
2020-03-18 13:31:41.200  INFO 13848 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2020-03-18 13:31:41.200  INFO 13848 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2020-03-18 13:31:43.865  INFO 13848 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-03-18 13:31:44.268  INFO 13848 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 309ms. Found 4 JPA repository interfaces.
2020-03-18 13:31:45.936  INFO 13848 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2020-03-18 13:31:47.642  INFO 13848 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-03-18 13:31:47.643  INFO 13848 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.31]
2020-03-18 13:31:53.658  INFO 13848 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-03-18 13:31:53.659  INFO 13848 --- [restartedMain] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 12458 ms
2020-03-18 13:31:54.159  INFO 13848 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-03-18 13:31:54.277  INFO 13848 --- [restartedMain] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.12.Final
2020-03-18 13:31:54.548  INFO 13848 --- [restartedMain] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-03-18 13:31:54.975  INFO 13848 --- [restartedMain] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2020-03-18 13:31:55.100 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:55.108 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:55.108  INFO 13848 --- [Druid-ConnectionPool-Create-1427088974] c.a.druid.pool.DruidAbstractDataSource   : {dataSource-1} failContinuous is true
2020-03-18 13:31:55.616 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:56.119 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:56.622 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:57.128 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:31:57.632 ERROR 13848 --- [Druid-ConnectionPool-Create-1427088974] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:40.266  INFO 9496 --- [restartedMain] com.ykcoding.BlogApplication             : Starting BlogApplication on LAPTOP-3VL4VRID with PID 9496 (E:\coding\JavaFile\blog\target\classes started by Lenovo in E:\coding\JavaFile\blog)
2020-03-18 13:32:40.274 DEBUG 9496 --- [restartedMain] com.ykcoding.BlogApplication             : Running with Spring Boot v2.2.5.RELEASE, Spring v5.2.4.RELEASE
2020-03-18 13:32:40.275  INFO 9496 --- [restartedMain] com.ykcoding.BlogApplication             : The following profiles are active: dev
2020-03-18 13:32:40.352  INFO 9496 --- [restartedMain] o.s.b.devtools.restart.ChangeableUrls    : The Class-Path manifest attribute in C:\Users\Lenovo\.m2\repository\org\glassfish\jaxb\jaxb-runtime\2.3.2\jaxb-runtime-2.3.2.jar referenced one or more files that do not exist: file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.xml.bind-api-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/txw2-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/istack-commons-runtime-3.0.8.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/stax-ex-1.8.1.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/FastInfoset-1.2.16.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.activation-api-1.2.1.jar
2020-03-18 13:32:40.352  INFO 9496 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2020-03-18 13:32:40.353  INFO 9496 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2020-03-18 13:32:41.849  INFO 9496 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-03-18 13:32:41.953  INFO 9496 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 95ms. Found 4 JPA repository interfaces.
2020-03-18 13:32:43.239  INFO 9496 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2020-03-18 13:32:43.250  INFO 9496 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-03-18 13:32:43.251  INFO 9496 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.31]
2020-03-18 13:32:43.802  INFO 9496 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-03-18 13:32:43.803  INFO 9496 --- [restartedMain] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 3450 ms
2020-03-18 13:32:44.276  INFO 9496 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-03-18 13:32:44.425  INFO 9496 --- [restartedMain] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.12.Final
2020-03-18 13:32:45.131  INFO 9496 --- [restartedMain] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-03-18 13:32:45.442  INFO 9496 --- [restartedMain] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2020-03-18 13:32:45.476 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:45.484 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:45.498  INFO 9496 --- [Druid-ConnectionPool-Create-1931535988] c.a.druid.pool.DruidAbstractDataSource   : {dataSource-1} failContinuous is true
2020-03-18 13:32:46.291 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:46.794 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:47.297 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:47.800 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:48.302 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:32:48.805 ERROR 9496 --- [Druid-ConnectionPool-Create-1931535988] com.alibaba.druid.pool.DruidDataSource   : create connection SQLException, url: jdbc:mysql://localhost:3306/blogService, errorCode 1049, state 42000

com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: Unknown database 'blogservice'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:885)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3421)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1247)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2775)
	at com.mysql.jdbc.Connection.<init>(Connection.java:1555)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1643)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1709)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2715)

2020-03-18 13:33:30.682  INFO 10432 --- [restartedMain] com.ykcoding.BlogApplication             : Starting BlogApplication on LAPTOP-3VL4VRID with PID 10432 (E:\coding\JavaFile\blog\target\classes started by Lenovo in E:\coding\JavaFile\blog)
2020-03-18 13:33:30.686 DEBUG 10432 --- [restartedMain] com.ykcoding.BlogApplication             : Running with Spring Boot v2.2.5.RELEASE, Spring v5.2.4.RELEASE
2020-03-18 13:33:30.687  INFO 10432 --- [restartedMain] com.ykcoding.BlogApplication             : The following profiles are active: dev
2020-03-18 13:33:30.833  INFO 10432 --- [restartedMain] o.s.b.devtools.restart.ChangeableUrls    : The Class-Path manifest attribute in C:\Users\Lenovo\.m2\repository\org\glassfish\jaxb\jaxb-runtime\2.3.2\jaxb-runtime-2.3.2.jar referenced one or more files that do not exist: file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.xml.bind-api-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/txw2-2.3.2.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/istack-commons-runtime-3.0.8.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/stax-ex-1.8.1.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/FastInfoset-1.2.16.jar,file:/C:/Users/Lenovo/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.2/jakarta.activation-api-1.2.1.jar
2020-03-18 13:33:30.834  INFO 10432 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2020-03-18 13:33:30.835  INFO 10432 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2020-03-18 13:33:32.634  INFO 10432 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-03-18 13:33:32.797  INFO 10432 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 153ms. Found 4 JPA repository interfaces.
2020-03-18 13:33:34.880  INFO 10432 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2020-03-18 13:33:34.899  INFO 10432 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-03-18 13:33:34.899  INFO 10432 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.31]
2020-03-18 13:33:35.278  INFO 10432 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-03-18 13:33:35.279  INFO 10432 --- [restartedMain] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 4444 ms
2020-03-18 13:33:35.911  INFO 10432 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-03-18 13:33:36.010  INFO 10432 --- [restartedMain] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.12.Final
2020-03-18 13:33:36.131  INFO 10432 --- [restartedMain] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-03-18 13:33:36.287  INFO 10432 --- [restartedMain] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2020-03-18 13:33:36.648  INFO 10432 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL55Dialect
2020-03-18 13:33:38.630  INFO 10432 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-03-18 13:33:38.642  INFO 10432 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-03-18 13:33:38.844  INFO 10432 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-03-18 13:33:39.796  WARN 10432 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-03-18 13:33:40.234  INFO 10432 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2020-03-18 13:33:40.490  INFO 10432 --- [restartedMain] o.s.b.a.w.s.WelcomePageHandlerMapping    : Adding welcome page template: index
2020-03-18 13:33:41.724  INFO 10432 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2020-03-18 13:33:41.737  INFO 10432 --- [restartedMain] com.ykcoding.BlogApplication             : Started BlogApplication in 12.05 seconds (JVM running for 14.778)
